packages/slate-editor/src/modules/editor-v4-rich-formatting/components/BlockQuoteElement/BlockQuoteElement.scss

```diff
@@ -1,11 +1,53 @@
 @import "styles/variables";
 
 .editor-v4-block-quote-element {
-    margin: $spacing-3 0;
-    padding: 20px;
-    border-left: 4px solid #c4c4c4;
-    color: $legacy-color-type-secondary;
-    font-size: $editor-v4-paragraph-font-size;
-    line-height: $editor-v4-paragraph-line-height;
-    font-style: italic;
+    padding: $spacing-3;
+    background-color: $grey-50;
+    max-width: fit-content;
+    position: relative;
+
+    &__paragraph {
+        padding: 0 $spacing-2;
+        background-color: $grey-50;
+        margin: 0;
+        line-height: 160%;
+        font-weight: 500;
+        min-width: 3em;
+
+        &--alignLeft {
+            border-left: 2px solid $grey-400;
+            text-align: left;
+        }
+
+        &--alignCenter {
+            border-left: 2px solid $grey-400;
+            text-align: center;
+        }
+
+        &--alignRight {
+            border-right: 2px solid $grey-400;
+            text-align: right;
+        }
+    }
+
+    &__paragraph::before {
+        content: attr(data-placeholder);
+        color: $grey-400;
+        position: absolute;
+    }
+
+    &--alignLeft {
+        margin-left: 0 !important;
```

Comment:
@e1himself @aspirisen I am not proud of this `!important`. But I didn't find another way for that one, cause `.editor-v4 [data-slate-editor=true] > * ` was overwritten it always to auto. Maybe you have a better idea for that one :)

Reply:
This looks broken in the Story Editor: 

![Screenshot-2022-05-09,13-38-29](https://user-images.githubusercontent.com/370680/167393768-a1af4e75-d18a-4baf-ac62-d121497fd246.png)